Tryouts/Messaging-JS/src/client/MessageRouterClient.spec.ts

```diff
@@ -0,0 +1,597 @@
+import { MessageRouterClient, ClientState, Connection, OnMessageCallback, OnErrorCallback, OnCloseCallback } from ".";
+import { MessageContext, MessageHandler, MessageScope, TopicMessage, TopicSubscriber } from "..";
+import * as messages from "../protocol/messages";
+import "jest-extended"
+import { DuplicateEndpointError, ErrorTypes, MessageRouterError } from "../exceptions";
+import { Error } from "../protocol";
+
+describe("MessageRouterClient", () => {
```

Comment:
Nitpick but you have a lot of empty new lines in tests.

Reply:
That's intentional to improve readability.